problems/5.loops.js

```diff
@@ -2,17 +2,35 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
-
+// function evenNumbers() {}
+const evenNumbers = num => {
+  let evenNumArray = []
+  let i = 22
+  while (i > 0) {
+    if(i % 2 === 0) {
+     evenNumArray.push(i)
+    }
+      i++
+  }
+    return evenNumArray
 }
 
 /** Returns all multiples of a given number up to 1000
  * Example if num was 3 the output would be [0, 3, 6, 9, 12, ..., 999]
 * @param {number} num - The number to find multiples of
 * @returns {number[]} - An array of all multiples of {num} up to 1000
 */
-function multiplesToOneThousand() {
-
+// function multiplesToOneThousand() {
+const multiplesToOneThousand = num => {
+  let multipleArray = []
+  let i = 0
+  while (i <= 1000) {
+    if(i % 5 === 0) {
```

Comment:
why mod 5 ? Are you looking for things only divisible from 5 ? Should be by num. Or better increment i by num.